### ISSUE TITLE
[Bug]: aria-describedby not working with <Link> tag

### ISSUE THREAD

**Issue by agravell047** [remix-run/react-router#11606]
### What version of React Router are you using?

3 + react-router-dom 5.3.0

### Steps to Reproduce

Add aria-describedby to a `<Link>` tag, notice that it doesn't announce the label using Mac VO. Switch to using an `<a>` tag, see that it is announced.

### Expected Behavior

The tag would announce the label referenced by aria-describedby the same way an `<a>` tag would.

### Actual Behavior

When trying to use aria-describedby with the `<Link>` tag to give users more context about the link, it is not announced when using this version of RR. I was forced to switch to using an `<a>` tag which works as expected.
